.github/workflows/docker.yml

```diff
@@ -1,50 +1,36 @@
-name: Docker build
-on:
-  pull_request:
-    types: [opened, synchronize, reopened]
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-      - name: Login to DockerHub
-        uses: docker/login-action@v2
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-      # branch names with special characters cannot be used to create docker image tag,
-      # so removing all special characters and replacing them with - for valid docker image tag.
-      - name: Modify Branch Name
-        run: echo "MODIFIED_BRANCH_NAME=$(echo '${{ github.ref_name }}' | sed 's/[^a-zA-Z0-9]/-/g')" >> $GITHUB_ENV
-      - name: Print Modified Branch Name
-        run: echo "Modified the branch name to $MODIFIED_BRANCH_NAME"
-      - name: Build and Push carma-messenger-ui Docker Image
-        uses: docker/build-push-action@v3
-        with:
-          context: ./carma-messenger-ui/
-          push: true
-          tags: usdotfhwastol/carma-messenger-ui:${{ env.MODIFIED_BRANCH_NAME }}
-
-      - name: Build and Push carma-messenger-core Docker Image
-        uses: docker/build-push-action@v3
-        with:
-          context: ./carma-messenger-core/
-          push: true
-          tags: usdotfhwastol/carma-messenger-core:${{ env.MODIFIED_BRANCH_NAME }}
-
-      - name: Build and Push chevrolet_tahoe_2018 Docker Image
-        uses: docker/build-push-action@v3
-        with:
-          context: ./carma-messenger-config/chevrolet_tahoe_2018/
-          push: true
-          tags: usdotfhwastol/carma-messenger-config:${{ env.MODIFIED_BRANCH_NAME }}-chevrolet_tahoe_2018
-
-      - name: Build and Push development Docker Image
-        uses: docker/build-push-action@v3
-        with:
-          context: ./carma-messenger-config/development/
-          push: true
-          tags: usdotfhwastol/carma-messenger-config:${{ env.MODIFIED_BRANCH_NAME }}-development
+name: Docker build
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and Push carma-messenger-ui Docker Image
+        uses: docker/build-push-action@v3
```

Comment:
Can we update to the latest `build-push-action` version? This affects multiple places.